config.js

```diff
@@ -179,11 +179,6 @@ var conf = convict({
       format: String,
       default: 'https://app.adjust.com/2uo1qc?campaign=fxa-conf-email&adgroup=ios&creative=button&fallback=https%3A%2F%2Fitunes.apple.com%2Fapp%2Fapple-store%2Fid989804926%3Fpt%3D373246%26ct%3Dadjust_tracker%26mt%3D8'
     },
-    signInUrl: {
```

Comment:
Nice catch, thanks.  I can't find any references to `signInUrl` in puppet-config so I think we're safe there.